notebooks/official/automl/sdk_automl_tabular_regression_batch_bq.ipynb

```diff
@@ -0,0 +1,968 @@
+{
```

Comment:
<p> </p><p> </p><h3>Create and run training pipeline </h3><p> </p><p>To train an AutoML model, <strong>create and run a training pipeline.</strong></p><p> </p><h4>Create training pipeline </h4><p> </p><p><strong>Create an AutoML training pipeline using the </strong><code><strong>AutoMLTabularTrainingJob</strong></code><strong> class, with the following parameters: </strong></p><ul><li><code>display_name</code>: The human readable name for the&nbsp;<code>TrainingJob</code>&nbsp;resource</li><li><code>optimization_prediction_type</code>: The task type for which to train the model</li><li>  - <code>classification</code>: A <strong>tabular</strong> classification model. &nbsp;  - <code>regression</code>: A tabular regression model.</li><li><code>column_transformations</code>: (Optional): Transformations to apply to the input columns</li><li><code>optimization_objective</code>: The optimization objective<strong> (minimize or maximize)</strong></li><li>  - binary classification: </li><li class="ql-indent-1"><code>minimize-log-loss</code></li><li class="ql-indent-1"><code>maximize-au-roc</code></li><li class="ql-indent-1"><code>maximize-au-prc</code></li><li class="ql-indent-1"><code>maximize-precision-at-recall</code></li><li><code>maximize-recall-at-precision</code></li><li class="ql-indent-1">  - multi-class classification: </li><li><code>minimize-log-loss</code></li><li class="ql-indent-1">  - regression: </li><li class="ql-indent-1"><code>minimize-rmse</code></li><li class="ql-indent-1"><code>minimize-mae</code></li><li class="ql-indent-1"><code>minimize-rmsle</code></li></ul><p>The instantiated object is the DAG (directed acyclic graph) for the training pipeline.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Done.</p>